crosswalks/csdms.csv

```diff
@@ -0,0 +1,74 @@
+Property,csdms
+codeRepository,Source web address
+programmingLanguage,Programming language / Other Programming language
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,module_type
+applicationSubCategory,
+downloadUrl,
+fileSize,
+installUrl,
+memoryRequirements,Memory requirements
+operatingSystem,Supported platforms / Other platform
+permissions,
+processorRequirements,Code optimized for (single/multiple processors)
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,
+storageRequirements,
+supportingData,
+author, First name / Last name
```

Comment:
"First name / Last name" are already mapped to givenName/familyName, no need to map them to "author" too.